ffi/src/lib.rs

```diff
@@ -959,6 +974,106 @@ pub unsafe extern "C" fn command(
     })
 }
 
+#[repr(C)]
+pub struct Cmder {
+    request_type: RequestType, // Equivalent to Go's C.RequestType
+    args_count: c_ulong,
+    args: *const *const c_char, // Array of C strings
+}
+
+#[repr(C)]
+pub struct Transaction {
+    cmd_count: c_ulong,
+    commands: *const Cmder,
+}
+
+/// Executes a command.
```

Comment:
```suggestion
/// Executes a transaction.
```
Please add a few more lines, since, this function is present in command ffi layer.

Reply:
Added new comments in Lib.rs.